easybuild/easyconfigs/s/snpEff/snpEff-5.0-GCCcore-9.3.0-Java-13-Python-3.8.2.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'Tarball'
+
+name = 'snpEff'
+version = '5.0'
+versionsuffix = '-Java-%(javaver)s-Python-%(pyver)s'
+
+homepage = 'https://pcingola.github.io/SnpEff/'
+description = """SnpEff is a variant annotation and effect prediction tool.
+ It annotates and predicts the effects of genetic variants (such as amino acid changes)."""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+source_urls = ['https://snpeff.blob.core.windows.net/versions/']
+sources = ['%%(name)s_v%s_core.zip' % version.replace('.', '_')]
+checksums = ['2b177440eb4a2c3bbfc8f50d4adf9e405f3dcfc05a31a55c96f773f07832af34']
+
+dependencies = [
+    ('Java', '13', '', True),
```

Comment:
For most of the other Java based software in GCCcore 9.3.0 (and 2020a) we are using Java 11. If this module is likely to be loaded at the same time as other modules, and snpEff works with Java 11, then it would be good to stick to that Java version.

Reply:
according to their website, it requires Java 12+

Reply:
Thanks for the clarification on this.